apps/framework-cli-e2e/test/backward-compatibility.test.ts

```
@@ -189,18 +189,26 @@ async function verifyTypeScriptVersionsMatch(
+ * from. Pinning both to the CLI's actual version guarantees a self-consistent
+ * n-1 setup.
  */
-function enforceLatestTypeScriptDependencies(projectDir: string): void {
+function enforceLatestTypeScriptDependencies(
+  projectDir: string,
+  cliVersion: string,
+): void {
   const packageJsonPath = path.join(projectDir, "package.json");
   const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf-8"));
 
   const dependencies = (packageJson.dependencies ??= {});
```

Biome (2.4.15)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)